tests/unittest/_torch/speculative/test_eagle3.py

```diff
@@ -141,8 +141,7 @@ def test_llama_eagle3(use_cuda_graph: bool, attn_backend: str,
     kv_cache_config = KvCacheConfig(enable_block_reuse=enable_block_reuse,
                                     max_tokens=8192)
     cuda_graph_config = CudaGraphConfig(
-        batch_sizes=[i for i in range(1, max_batch_size +
-                                      1)]) if use_cuda_graph else None
+        max_batch_size=max_batch_size) if use_cuda_graph else None
```

Comment:
the 2 changes aren't equivalent right? the granularity is 64 by default if we simply specify `max_batch_size` - but the prior change is explicitly enumerating batch_sizes with granularity 1 right?

Reply:
@venkywonka , no, the key is you cannot set `max_batch_size` and `batch_sizes` at same time ..

Reply:
and this test is not for perf test, so I keep [1,2,4] instead of [1,2,3,4]